src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md

```diff
@@ -0,0 +1,274 @@
+        },
+      },
+    },
+  }
+}
+```
+
+</CodeGroup>
+
+Previously, users were required to handle the creation and closing of the browser instance using methods like `chromium.launch()` and `chromium.connect()` for creation, and the `browser.close()` method for releasing resources before ending the iteration. This repetitive code has been eliminated.
+
+Now, all that is needed is to specify the browser type within the [scenario options](#scenario-options). A browser instance will be automatically created and closed for each iteration by the browser module, streamlining the process.
+
+This change helps identify the test as a browser test and allows automatic control of the browser's lifecycle. Users no longer need to start or stop the browser manually through the browser API. If the `browser.type` option is set in the scenario options, a browser instance will automatically start at the beginning and close at the end of each test iteration.
+
+
+## Opening a new page
+
+You can open a new page by using the imported [browser](/javascript-api/k6-experimental/browser/#browser-module-api) object's [browser.newPage()](/javascript-api/k6-experimental/browser/newpage) method. You can still using the [Page](/javascript-api/k6-experimental/browser/page/) object as before.
+
+<CodeGroup labels={["After"]} lineNumbers={[true]}>
+
+<!-- eslint-skip -->
+
+```javascript
+export default async function () {
+    const page = browser.newPage();
+    // ...
+    page.close();
+}
+```
+</CodeGroup>
+
+<Blockquote mod="note" title="">
+
+Closing of the page is critical for the calculation of accurate Web Vital metrics. See the [browser metrics](/using-k6-browser/browser-metrics/) for more details.
+
+</Blockquote>
+
+
+
+
+## Simplified resource management
+
+As [mentioned earlier](#scenario-options), in the new API, there's a shift in how the browser's lifecycle is managed.
+
+Since the browser lifecycle is automatically managed by the browser module, the closing of the browser has been simplified. The explicit `browser.close()` call has been removed. Simply close the page using the [page.close()](/javascript-api/k6-experimental/browser/page/close/) method as in the example below.
+
+<CodeGroup labels={["Before"]} lineNumbers={[true]}>
+
+<!-- eslint-skip -->
+
+```javascript
+page.close();
+browser.close();
+```
+
+</CodeGroup>
+
+<CodeGroup labels={["After"]} lineNumbers={[true]}>
+
+<!-- eslint-skip -->
+
+```javascript
+page.close();
+```
+
+</CodeGroup>
+
+
+
+
+## Browser context limit
+
+Since the browser lifecycle is now being managed by the [browser module](/javascript-api/k6-experimental/browser/), the new browser implementation limits the use to a single active [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) per iteration. This change supports better prediction of resource requirements for a test run and allows for more controlled management of [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/)s.
```

Comment:
Should we mention here something about scenarios?. E.g.:
```suggestion
Since the browser lifecycle is now being managed by the [browser module](/javascript-api/k6-experimental/browser/), the new browser implementation limits the use to a single active [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) per iteration. This change supports better prediction of resource requirements for a test run, allows for more controlled management of [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/)s and promotes [scenarios](TODO:link) as the way to separate independent browser sessions.
```
cc: @mdcruz

Reply:
For this, I've applied Ankur's suggestion.